tpdatasrc/tpgamefiles/scr/tpModifiers/extra_music.py

```diff
@@ -0,0 +1,23 @@
+#Extra Music: SRD
+
+from templeplus.pymod import PythonModifier
+from toee import *
+import tpdp
+
+print "Registering Extra Music"
+
+def EMNewDay(attachee, args, evt_obj):
+
+	BardicMusicCount = attachee.has_feat("Extra Music")
+
+
+	#Extra Music grants 4 additional uses of Bardic Music each time the feat is taken
+  #Max music per day remains 3, can't find a way to alter it. Works properly nonetheless.
```

Comment:
If you mean the max on the radial menu, currently it just takes bard level (condition.cpp BardicMusicRadial). I suppose you can change it to max(bardLvl, arg0). You can also generalize it to take the value from a query if you anticipate modification by other feats.